main.go

```diff
@@ -63,7 +63,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.Parse()
 
-	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
+	ctrl.SetLogger(zap.New(zap.UseDevMode(false)))
```

Comment:
👍  This changes logging format in positive way (especially for use with logging solutions like ELK), but it would be good to have this change in a separate PR, in order to make it visible in release history, plus it's a bit out of scope of this PR.